.buildkite/pipeline.yml

```diff
@@ -0,0 +1,47 @@
+steps:
+  - label: ":rubocop: Rubocop"
```

Comment:
Does the current codebase pass both `ruboco` and `brakeman`? Else this PR would block future PRs

Reply:
Yup, it passes rubocop and RSpec, I didn't include brakeman, this could be the next step.

Reply:
https://github.com/returnly/event_tracker/pull/11 Instead of adding rubocop exceptions in rubocop-todo I preferred to fix them all.